fixtures/coverall/tests/bindings/test_coverall.kts

```diff
@@ -488,3 +509,7 @@ Coveralls("using_fakes_with_real_objects_crashes").use { coveralls ->
     }
     assert(exception != null)
 }
+
+// This is from an earlier GC test; ealier, we made 1000 new objects.
+// By now, the GC has had time to clean up, and now we should see 0 alive.
+assert(getNumAlive() == 0UL) { "Num alive is ${getNumAlive()}. GC/Cleaner hasn't run fully" };
```

Comment:
I guess there's no guarantee that the GC actually ran at this point, right? Will this lead to annoying intermittents on CI?

Reply:
Perhaps. On my slow machine, this test never failed. 

The test failure on first push (not this one) suggested that the thread was working rather better on CI than my machine.

The alternative: putting in ~500ms sleep– which I was trying to avoid.

Reply:
What about manually calling `System.gc()` before? It doesn't actually guarantee the GC will be done at that point, but it should make it more likely.